cmd/get.go

```diff
@@ -178,6 +180,19 @@ func GetDeploySpecTable(specs []deploymentspec.DeploymentSpec) (string, []string
 			spec.GetString("deployStrategy/type"),
 		)
 		rows = append(rows, row)
+		releaseToRows = append(releaseToRows, spec.GetString("releaseTo"))
+	}
+	if flagVerbose {
```

Comment:
Veldig elegant løsning for å injisere felter, men er den i overkant komplisert? 
Synes vi bør sjekke med bestiller om vi kan legge til ekstra felter på slutten.